subsys/bluetooth/controller/ll_sw/ull_connections.c

```diff
@@ -390,6 +393,28 @@ void ull_conn_done(struct node_rx_event_done *done)
 		return;
 	}
 
+	/* Master transmitted ack for the received terminate ind or
```

Comment:
I find this kind of if statements with interleaved #ifdefs really hard to read 
Also  (or therefore ;-)) I don't understand why the (reason == BT_HCI_ERR_TERM_DUE_TO_MIC_FAIL) check is under CENTRAL-def when comment says slave received MIC failure

Reply:
> I find this kind of if statements with interleaved #ifdefs really hard to read
> Also (or therefore ;-)) I don't understand why the (reason == BT_HCI_ERR_TERM_DUE_TO_MIC_FAIL) check is under CENTRAL-def when comment says slave received MIC failure

This code is copied directly from the legacy ull_conn.c and i haven't looked into why it's written like that. This code is not related to LLCP functionality but is existing connection logic that was left out during the creation of the ull_connections.c file.